src/components/global/Task/Task.jsx

```diff
@@ -12,20 +12,25 @@ function Task(props) {
   const handleIconClick = () => setIsExpanded(!isExpanded);
 
   return (
-    <div className={"task-container" + (isExpanded ? "--expanded" : "")}>
+    <div className={`task-container${  isExpanded ? "--expanded" : ""}`}>
       <div className="task-header-container">
         <div
-          className={"task-checkbox" + (isChecked ? "--checked" : "")}
+          className={`task-checkbox${  isChecked ? "--checked" : ""}`}
           onClick={handleCheckBoxClick}
+          onKeyDown={handleCheckBoxClick}
```

Comment:
why do we need these fields?

Reply:
OnKeyDown is from this linting rule: [click-events-have-key-events](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md)